### ISSUE TITLE
Execution problem

### ISSUE THREAD

**Issue by ellalalalalalala** [jumphone/SPRINT#29]
Hi :)

I´m trying to run SPRINT via script, not from command line. The folder, I start the script in, contains sprint, bwa and samtools as subfolders. I have two questions.  

1) When I´m trying to run sprint prepare and sprint main via script I get the error:
/DIRECTION/sprint-0.1.7/sprint: Is a directory

In a previous issue, I found, that you suggested to run "chmod 777 sprint", which I ran in the sprint folder, but it didn´t resolve the problem.

Do you have an idea, what went wrong or what I could do to resolve this issue?

2) The Github page suggests, that there is "sprint_from_bam.py" in the main folder of sprint ("sprint-0.1.7"). I don´t find it in my installed folder. How could I add or execute it?

Thanks a lot in advance!

**Comment by jumphone** [comment]
1. Please check wether you can run /DIRECTION/sprint-0.1.7/sprint from command line.

2. You can download it from: https://github.com/jumphone/SPRINT/blob/master/sprint_from_bam.py

**Comment by ellalalalalalala** [comment]
Thanks for your fast answer! :)

1. I moved to the directory of the main sprint folder ("sprint-0.1.7"), established the python 2.7.8 environment and tried to run the "sprint prepare" command. Didn´t work ("-bash: sprint: command not found")! So probably something with my configuration wrong? Everything seemed fine when I followed the installation instructions.

2. Great, I will do that, thank you!
